src/main/jbake/content/documentation/bundles/org-apache-sling-junit-bundles.md

```diff
@@ -11,7 +11,7 @@ provide different examples including HTTP-based and server-side teleported tests
 bundle module, running against a full Sling instance setup in the same Maven module.
 
 ## org.apache.sling.junit.core: server-side JUnit tests support
-This bundle provides a `JUnitServlet` that runs JUnit tests found in bundles. 
+This bundle provides a `JUnitServlet` that runs JUnit tests found in bundles. Both JUnit 4 tests and (optionally) JUnit 5 tests (aka Jupiter) are supported.
```

Comment:
Not sure if the spelling should by "Junit4" or "Junit 4" with a space.

The former returns 4x more results on google.com and has my preference in terms of searching for these things.

No big deal - feel free to use whatever works for you!

Reply:
In the JUnit 5 documentation they use the spaced spelling for both JUnit 4 and JUnit 5. That's why I chose it. Also I tried to be consistent within this document, I hope I've achieved that :)